### PR TITLE
add multiarch support to manual approval gate feature

### PR DESCRIPTION
**Pull request by Basavaraju-G**
This PR will add support for s390x,ppc64le multi architecture support.
https://github.com/openshift-pipelines/manual-approval-gate/issues/168

### CODE REVIEW

File: .github/workflows/latest.yml [hunk 1]
```diff
@@ -88,4 +88,4 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
     - uses: imjasonh/setup-ko@v0.7
     - run: |
-        ko publish --push=${{ github.event_name != 'pull_request' }} --base-import-paths ./cmd/controller
```
Comment (piyush-garg) [inline .github/workflows/latest.yml:91]: should we add arm also?

Comment (Basavaraju-G) [reply .github/workflows/latest.yml:91]: okay @piyush-garg i will add it.

Comment (Basavaraju-G) [reply .github/workflows/latest.yml:91]: > should we add arm also?

Added.

**Pull request merged**